packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -4,58 +4,250 @@
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
 import cx from 'classnames';
-import { Popover, Box } from '../utils/index.js';
-import type { CommonProps, PopoverProps } from '../utils/index.js';
+import {
+  useFloating,
+  autoUpdate,
+  offset,
+  flip,
+  shift,
+  useClick,
+  useHover,
+  useFocus,
+  useDismiss,
+  useRole,
+  useInteractions,
+  safePolygon,
+  size,
+  autoPlacement,
+  hide,
+  inline,
+} from '@floating-ui/react';
+import type { Placement } from '@floating-ui/react';
+import { Box, getDocument, useGlobals, useMergedRefs } from '../utils/index.js';
+import type { PolymorphicForwardRefComponent } from '../utils/index.js';
+import ReactDOM from 'react-dom';
 
-export type TooltipProps = {
+type TooltipOptions = {
+  /**
+   * Placement of the Tooltip
+   * @default 'top'
+   */
+  placement?: Placement;
+  /**
+   * Property for manual visibility control
+   */
+  visible?: boolean;
+  /**
+   * autoUpdate options
+   * https://floating-ui.com/docs/autoUpdate#options
+   */
+  updateOptions?: {
```

Comment:
should be called `autoUpdateOptions` maybe?

Reply:
Updated

Reply:
> By default everything is false.

This is not true. Some of these are true by default in floating-ui. https://floating-ui.com/docs/autoupdate#options

Also missing `layoutShift` option.

Reply:
Oops, added

Reply:
Update seems to fix the issue